compose.go

```diff
@@ -103,8 +104,16 @@ func NewLocalDockerCompose(filePaths []string, identifier string, opts ...LocalD
 
 	dc.absComposeFilePaths = make([]string, len(filePaths))
 	for i, cfp := range dc.ComposeFilePaths {
-		abs, _ := filepath.Abs(cfp)
-		dc.absComposeFilePaths[i] = abs
+		if cfp == "-" {
```

Comment:
Mmm I do not get this line. Is it expected to receive a `-` as a file path for a compose file?

Reply:
Yes, it is